pkgs/by-name/ca/casadi/package.nix

```diff
@@ -30,7 +30,7 @@
   #sundials,
   superscs,
   spral,
-  swig,
```

Comment:
Also this change looks logically separate from the Dawrin build change.

Reply:
> it is not: swig 3 generated code contains things in C++ std which were removed in C++17, so it won't build on darwin. But you're totally right: it looks like it is separate, so I'm adding a comment to document that.

Still though, if it builds fine with swig 4 on Linux as well, it can be considered a separate change, as even Linux users will want to know that the latest and greatest swig generated the C++ code that is part of this package.